data/hai/hai reveal 1 intro.txt

```diff
@@ -309,15 +307,66 @@ mission "ZZ Event Hai Secret Leaks: FW"
 		event "news: hai video clip published"
 		event "pirates investigate leaks"
 		conversation "hai reveal free worlds news"
-	
-	
+
+
 event "pirates investigate leaks"
 	system "Waypoint"
 		government "Pirate"
 		add fleet "Marauder fleet IX" 2000
 		add fleet "Anti-Pirate Hai (No Wormhole Travel)" 700
 
 
+mission "Trader Directions to Hai Space"
+	landing
+	name "Investigate the Hai"
+	description "Follow the coordinates given to you by Captain Casey and meet the Hai."
+	source Sunracer
+	waypoint "Ultima Thule"
+	destination "Allhome"
+	to offer
+		or
+			has "ZZ Event Hai Secret Leaks: RS: offered"
+			has "ZZ Event Hai Secret Leaks: FW offered"
+	on offer
+		conversation
+			`Arriving at the only place you can think of where you feel sure you can find out more information, you find that here, like most places, is abuzz with the recent news. Avoiding the many large, cheap, and boisterous establishments that service the many crews and workers of the Syndicate corporate machinery here on Sunracer, you make your way to one of the marginally more up-market places, cunningly marketed as a merchant-only dive.`
+			`	The endless cycle of both the news and government propaganda being displayed on most screens in major thoroughfares is remarkably absent here, with only one of the screens on display covering the "violent alien race" at all - running stats on how many different versions of the story are now being told.`
+			`	Casting around the room you pick out a merchant who looks like they've been around the block a few times. In addition to being a woman who looks like she could pick up most people and break them over her knee, she's conspicuous by the threatening glare you watch her give a younger captain who left a table talking about the Hai to approach her... briefly. As you make your way over though you see her glare falter in a flash of recognition, before subsiding to a reserved scowl.`
+			`	She waits until you're directly next to her table. "The famous Captain <last>. Which ancient Sumerian deity did I offend to be granted the gift of your attention hey?"`
+			`	The emphasis she places on the word "gift" is dripping in sarcasm, but her overall tone is wary rather than hostile, and she gestures without comment to the seat opposite for you to sit.`
+			choice
+				`	"I'm just here to find out more about the Hai."`
+					goto innocent
+				`	"You looked to be the most experienced person in the room."`
+					goto practical
+				`	"Why ask a rookie, when there's a professional available?"`
+			`	"Pff," she scoffs, but despite rolling her eyes a slight smirk undermines her otherwise wary expression. "Flattery from a hero. You've been hanging out with too many politicians <last>."`
```

Comment:
```suggestion
			`	She scoffs, but despite her rolling eyes, a slight smirk undermines her otherwise wary expression. "Flattery from a hero. You've been hanging out with too many politicians, <last>."`
```

Reply:
Repositioning the word rolling feels very off to me.